archive/l/lobster/helloworld.lobster

```diff
@@ -0,0 +1,10 @@
+import std
+import vec
+import color
+
+fatal(gl_window("Shooter Tutorial", 640, 480))
```

Comment:
Does this launch a window with that title? If so, maybe we could try "Hello World" or something similar.

Reply:
Yes I'll change the title.